workflow/commit-messages.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Commit guidelines
+description: Some general guidelines to help you in writing good commits for collaboration and clarity.
+---
+
+This document was written as an overview of how an optimal commit should be written. But in the end it is still your decision , no Pull Request should be stuck in a review phase because its commits differ from the practices described below.
```

Comment:
```suggestion
This document was written as an overview of how an optimal commit should be written. But in the end it is still your decision, no Pull Request should be stuck in a review phase because it's commits differ from the practices described below.
```